### PR TITLE
[add] Homepage component

### PR DESCRIPTION
**Pull request by edbella**
I've added the Homepage component which displays the navbar and some information about th e application.

### CODE REVIEW

**Comment by edbella** [comment]
@kechyy @Segun-Ade @justiceotuya @ChineduOguejiofor 
Following up on the previous PR... Because the homepage component is added, all checks passed.

File: src/components/homepage/components/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import { Link } from "react-router-dom";
+import { Row, Col } from "antd";
+import { Title } from "../constants";
+import Artwork from "../images/hero-illustration.svg";
+
+class Homepage extends Component {
+  render() {
+    return (
+      <>
+        <Helmet titleTemplate="%s - CleanIT" defaultTitle="CleanIT">
```
Comment (justiceotuya) [inline src/components/homepage/components/Homepage.jsx]: All strings should be in your constant files example 

**constant.js file**
export const STRINGS = {
TITLE_TEMPLATE : "%s - CleanIT",
TITLE_TEMPLATE_DEFAULT_TITLE : "CleanIT"
}

**Homepage.js file**
 <Helmet titleTemplate={TITLE_TEMPLATE} defaultTitle={TITLE_TEMPLATE_DEFAULT_TITLE } />



you should do this for all your strings

File: src/components/homepage/components/Homepage.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          <meta charSet="utf-8" />
+          <meta
+            name="description"
+            content="CleanIT is a on-demand laundry service provider that provides vendors to users, ensuring efficient and quality service from all vendor"
```
Comment (justiceotuya) [inline src/components/homepage/components/Homepage.jsx]: refer to the previous review

File: src/components/homepage/components/Homepage.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                    alt=""
+                  />
+                </Col>
+
+                <Col lg={12} data-aos="fade" data-aos-delay="1000">
+                  <Title className="blue-text hero-title">
+                    Let us help you worry-less about your laundry
+                  </Title>
+                  <p>
+                    CleanIT is your on-demand laundry service middle-man. As a
```
Comment (justiceotuya) [inline src/components/homepage/components/Homepage.jsx]: refer to previous review

File: src/components/homepage/components/homepage.scss [hunk 4]
```diff
@@ -0,0 +1,71 @@
+.hero-section {
+  /* position: relative;
```
Comment (justiceotuya) [inline src/components/homepage/components/homepage.scss]: remove all unused code

**Comment by enyeInc** [comment]
@edbella Actually your PR should not be more than 250 lines of changes. How can you expect anyone to review 1100+ lines of code. Close this PR and create smaller PR from it

**Pull request closed without merging**